### ISSUE TITLE
ChoiceGroup focus conflict when it's the first component inside a focus zone

### ISSUE THREAD

**Issue by liyatilahun** [microsoft/fluentui#20173]
### Environment Information

- **Package version(s)**: 8.34.0
- **Browser and OS versions**: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen: https://codesandbox.io/s/focustrapzone-choicegroup-focus-conflict-9d96j?file=/src/index.tsx:79-92 (**Comment out the TextField inside the code to repro the issue**)

![choicegroup_focus_conflict](https://user-images.githubusercontent.com/92185632/136627630-903e9865-94ea-48fb-b76f-0e19d9dbcc2c.gif)

#### Actual behavior:
The focus on the radio button does not focus on the selected option, but instead focus on the first option of the choice group. This happens when using keyboard navigation (tab, keyUp and keyDown). But it works as expected for shift-tab. To repro this problem the ChoiceGroup should be the first component inside the focus zone.

#### Expected behavior:
The focus on the radio button should focus on the selected option instead of the first option when using keyboard navigation.

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No) No

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable) word-online

**Comment by gouttierre** [comment]
@liyatilahun  - Thanks for filing this issue with us. To set expectations the developers will review this issue once capacity allows. 

@khmakoto  - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by khmakoto** [comment]
Hi @gouttierre, @liyatilahun I can confirm that this is indeed an issue. I'll work on it once capacity allows.

**Comment by akvangari** [comment]
Hi, I believe I'm seeing the same issue for the `GroupedList` component. I'm unable to focus on elements after the first child. Here's a quick codesandbox: https://codesandbox.io/s/adoring-goldwasser-5ct2n
If this is a different issue, I can create a new one. Thanks

**Comment by spmonahan** [comment]
Hi @liyatilahun I've been looking into this issue a bit and I'd like to get a bit of context on where this issue manifests in your application. Where and how are you using the combination of `FocusTrapZone` and `ChoiceGroup`? Is it inside a dialog or another type of modal? Somewhere else?

**Comment by liyatilahun** [comment]
Hi Sean,

Thanks for reaching out about this issue! We were seeing this issue inside a dialog.
Please let me know if there is anything else you need clarification on.

Thanks,
Liya
________________________________

**Comment by spmonahan** [comment]
Thanks @liyatilahun ! I've merged a fix for this issue that has been released in v8.46.1. Can you pull this version into your app to verify the fix?

**Comment by liyatilahun** [comment]
@spmonahan  thank you! The issue has been fixed on my end. 😄
